From 77d18ece20f69ff1e1f6afd4b2d8cf2a1f252f3a Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Mon, 20 Sep 2021 03:46:51 -0400
Subject: [PATCH 1/2] disable sga device
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

RH-Author: Miroslav Rezanina <mrezanin@redhat.com>
RH-MergeRequest: 44: Apply RHEL 9.0.0 Beta fixes to RHEL 9.0.0
RH-Commit: [1/2] 3d8e1b51b496175de71162c612abbd64adbcb9e5 (mrezanin/centos-src-qemu-kvm)
RH-Bugzilla: 2000845
RH-Acked-by: Daniel P. Berrangé <berrange@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>

Disabling sga device we are going to not support anymore.

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 configs/devices/x86_64-softmmu/x86_64-rh-devices.mak | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configs/devices/x86_64-softmmu/x86_64-rh-devices.mak b/configs/devices/x86_64-softmmu/x86_64-rh-devices.mak
index 24b96ba0c4..20c2991941 100644
--- a/configs/devices/x86_64-softmmu/x86_64-rh-devices.mak
+++ b/configs/devices/x86_64-softmmu/x86_64-rh-devices.mak
@@ -67,7 +67,6 @@ CONFIG_SERIAL=y
 CONFIG_SERIAL_ISA=y
 CONFIG_SERIAL_PCI=y
 CONFIG_SEV=y
-CONFIG_SGA=y
 CONFIG_SMBIOS=y
 CONFIG_SMBUS_EEPROM=y
 CONFIG_TEST_DEVICES=y
-- 
2.27.0

